src/App.jsx

```diff
@@ -1,13 +1,372 @@
+// }
+//
+// return (
+//     <>
+//         <main>
+//         <h1>Fruitmand bezorgservice</h1>
+//             <h2>Jouw winkelmandje:</h2>
+//
+//             <Counter fruitObject={fruit} increment={fruitIncrement} decrement={fruitDecrement} fruitImageObject={fruitImage}/>
+//
+//
+//             {/*<div className="fruit-container">*/}
+//             {/*    <img src={aardbei} alt="aardbei afbeelding"/>*/}
+//             {/*    <h2>Aardbeien</h2>*/}
+//             {/*    <div className="fruit-button-container">*/}
+//             {/*    <button className="fruit-button" onClick={handleDecreaseStrawberry}>-</button>*/}
+//             {/*    <p>{quantityStrawberry}</p>*/}
+//             {/*    <button className="fruit-button" onClick={handleIncreaseStrawberry}>+</button>*/}
+//             {/*    </div>*/}
+//             {/*</div>*/}
+//
+//             {/*<div className="fruit-container">*/}
+//             {/*    <img src={banaan} alt="banaan afbeelding"/>*/}
+//             {/*    <h2>Bananen</h2>*/}
+//             {/*    <div className="fruit-button-container">*/}
+//             {/*    <button className="fruit-button" onClick={handleDecreaseBanana}>-</button>*/}
+//             {/*    <p>{quantityBanana}</p>*/}
+//             {/*    <button className="fruit-button" onClick={handleIncreaseBanana}>+</button>*/}
+//             {/*</div></div>*/}
+//
+//             {/*<div className="fruit-container">*/}
+//             {/*    <img src={appel} alt="appel afbeelding"/>*/}
+//             {/*    <h2>Appels</h2>*/}
+//             {/*    <div className="fruit-button-container">*/}
+//             {/*    <button className="fruit-button" onClick={handleDecreaseApple}>-</button>*/}
+//             {/*    <p>{quantityApple}</p>*/}
+//             {/*    <button className="fruit-button" onClick={handleIncreaseApple}>+</button>*/}
+//             {/*</div></div>*/}
+//
+//             {/*<div className="fruit-container">*/}
+//             {/*    <img src={kiwi} alt="kiwi afbeelding"/>*/}
+//             {/*    <h2>Kiwis</h2>*/}
+//             {/*    <div className="fruit-button-container">*/}
+//             {/*    <button className="fruit-button" onClick={handleDecreaseKiwi}>-</button>*/}
+//             {/*    <p>{quantityKiwi}</p>*/}
+//             {/*    <button className="fruit-button" onClick={handleIncreaseKiwi}>+</button>*/}
+//             {/*</div></div>*/}
+//             <button className="reset-button" type="button" onClick={resetForm}>Reset</button>
+//
+//             </main>
+//     </>
+//   )
+// }
+//
+// export default App
 
 import './App.css'
+import {useState} from "react";
+import { useForm } from 'react-hook-form';
+import Counter from "./components/Counter/Counter.jsx";
+import logo from "./assets/screenshot-logo.png";
+import Form from './components/Form/Form.jsx';
 
 function App() {
+    const initialFruit = {
+        apples: 0,
+        bananas: 0,
+        kiwis: 0,
+        strawberries: 0,
+    };
```

Comment:
Ik zou deze buiten het component zetten als deze nooit veranderd, anders krijg je bij elke render een nieuw object. Gaat opzich niets mis, maar is niet nodig 😉